testgrid/specs/os-full.yaml

```diff
@@ -54,6 +54,22 @@
     curl -L -o centos2ol.sh https://raw.githubusercontent.com/oracle/centos2ol/main/centos2ol.sh
     chmod +x centos2ol.sh
     bash centos2ol.sh -r
+- id: rhel-9
+  name: RHEL
+  version: "9.x"
+  vmimageuri: https://download.rockylinux.org/pub/rocky/9/images/x86_64/Rocky-9-GenericCloud-Base.latest.x86_64.qcow2
```

Comment:
can we not call rocky linux RHEL?

Reply:
This feels like it'll bite us eventually

Reply:
```suggestion
- id: rocky-9
  name: Rocky
  version: "9.x"
  vmimageuri: https://download.rockylinux.org/pub/rocky/9/images/x86_64/Rocky-9-GenericCloud-Base.latest.x86_64.qcow2
```

Reply:
updated